protos/flyteidl/plugins/spark.proto

```diff
@@ -21,4 +21,8 @@ message SparkJob {
     map<string, string> sparkConf = 4;
     map<string, string> hadoopConf = 5;
     string executorPath = 6; // Executor path for Python jobs.
+    // databricksConf is base64 encoded string which stores databricks job configuration.
```

Comment:
why is the databricks one a b64 encoded string while the spark and hadoop conf's above just a map of str, str?  Is the databricks one a binary object that needs to be encoded?

Reply:
Because it's nested dict. wait, I think I can use `Struct` here. Let me update it

Reply:
+1